### PR TITLE
Snapshot List Component

### PR DESCRIPTION
**Pull request by cutandpastey**
screen shot 2015-03-31 at 16 28 14

### CODE REVIEW

**Comment by jamespamplin** [comment]
`lib/icons/.DS_Store` checked in. Check your gitignores - should be in a system one really. Silly macos

File: lib/snapshot-list/index.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    restrict: 'E',
+    transclude: true,
+    replace: true,
+    template: snapshotList,
+    link: function(scope, el, attrs){
+      //loop through all children,
+      //calculate the time difference between each child
+      //assign it.
+      var children = el.find('ol').children();
+      for(var i = 0; i <= children.length; i++){
```
Comment (hmgibson23) [inline lib/snapshot-list/index.js:26]: I thought we hate for loops. I'm pretty indifferent... Don't we normally user `Array.prototype`?

File: lib/snapshot-list/snapshot-list.html [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<div class="snapshot-list">
+  <div class="snapshot-list__header">
+    <div class="snapshot-list__header__decal"><h4>No.</h4></div>
```
Comment (hmgibson23) [inline lib/snapshot-list/snapshot-list.html:3]: No.?

File: lib/snapshot-list/snapshot-list__item--active.html [hunk 3]
```diff
@@ -0,0 +1,20 @@
+<li>
+  <div class="snapshot-list__item--active">
+    <div class="snapshot-list__item__index">
+      <h4>{{ index }}</h4>
```
Comment (hmgibson23) [inline lib/snapshot-list/snapshot-list__item--active.html:4]: I'm probably wrong but won't putting spaces in between the {{ }} actually put the spaces in there? Swear I've seen that before.

File: lib/snapshot-list/snapshot-list.html (hunk 2 above)
Comment (cutandpastey) [inline lib/snapshot-list/snapshot-list.html:3]: Did you look at the jpeg in the description?

Comment (hmgibson23) [inline lib/snapshot-list/snapshot-list.html:3]: Apparently not :facepunch:

**Pull request closed without merging**